Remotion/Web/ClientScript/Scripts/DatePicker.ts

```diff
@@ -87,11 +94,11 @@ function DatePicker_CloseDatePicker(setFocusOnTarget)
   current.DatePicker.remove();
   if (current.TargetDocument != null)
   {
-    $(current.TargetDocument).unbind('click', current.ClickHandler);
+    $(current.TargetDocument).unbind('click', current.ClickHandler!); // TODO RM-7647: DatePick.DatePicker_CloseDatePicker removes all click handlers from target element if no click handler was specified for the DatePicker
```

Comment:
The assumtion was that `unbind` will only remove the specified `ClickHandler`, similiar to how C# events work. `current.ClickHandler` should not be null. I do see there's a mismatch with the bind-function. That's a bug.